classes/author.rb

```diff
@@ -0,0 +1,15 @@
+class Author
+  attr_reader :id, :firstname, :lastname
+  attr_accessor :items
+
+  def initialize(firstname, lastname)
+    @id = rand(1..1000)
+    @firstname = firstname
+    @lastname = lastname
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+  end
```

Comment:
To add the association relationship between `Author` and `Item` class you should assign the added author to the specific item, you can achieve that by adding something like this: 
`item.author = self`

You also should check the case where the item already exists to avoid adding double items with the same props, you can achieve that by modifying line 13 to be something like this: 
`items << item unless @items.include?(item)`